easybuild/tools/package/utilities.py

```diff
@@ -134,7 +135,7 @@ def package_with_fpm(easyblock):
         '-s', 'dir',  # source
         '--version', pkgver,
         '--iteration', pkgrel,
-        '--description', quote_str(easyblock.cfg["description"]),
+        '--description', quote_str(eb_shell_quote(easyblock.cfg["description"])),
```

Comment:
I'll check. I thought the quote_str would handle the outside wrapping, but it probably doesn't work the way I think it does.

The tests should help me figure it out I'd think.